packages/server/src/workers/subscription.ts

```diff
@@ -325,16 +329,16 @@ async function sendRestHook(
     logger.debug(`Ignore rest hook missing URL`);
     return;
   }
-
   const startTime = new Date().toISOString();
   const headers = buildRestHookHeaders(subscription, resource);
   const body = stringify(resource);
   let error: Error | undefined = undefined;
+  const fetchMethod = validDeleteInteraction(subscription) ? 'DELETE' : 'POST';
```

Comment:
Ooo.... interesting....  Is this in the R5 spec?  Surprising

Reply:
I don't see that in R5, I'll change this back